src/main/java/com/capool/backend/model/VehicleModel.java

```diff
@@ -0,0 +1,124 @@
+package com.capool.backend.model;
+
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+public class VehicleModel {
+
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column(name = "VEHICLE_TYPE", nullable = false)
+    private String vehicleType;
+
+    @Column(name = "VEHICLE_BRAND", nullable = false)
+    private String vehicleBrand;
+
+    @Id
+    @Column(name = "VEHICLE_NUMBER", nullable = false)
+    private String vehicleNumber;
+
+    @Column(name = "VEHICLE_MODEL", nullable = false)
+    private String vehicleModel;
+
+    @Column(name = "VEHICLE_COLOR", nullable = true)
+    private String vehicleColor;
+
+    @Column(name = "VEHICLE_CAPACITY", nullable = false)
+    private String vehicleCapacity;
+
+    @Column(name = "VEHICLE_OWNER", nullable = false)
+    private String vehicleOwner;
```

Comment:
Modify datatype as UserModel and map as One to One
**_Suggestion_**: Should we allow multiple owners of a vehicle? if so then datatype will be ```List<UserModel>``` with One to Many mapping

Reply:
mapped as ManyToOne
many vehicle with one user